manual/source/Tutorial/GettingStarted/ParsingCommandLineArguments.rst

```diff
@@ -639,3 +594,35 @@ Below, you can see a part of the rendered HTML and man pages generated by the co
 .. image:: modify_string.png
 
 For further deading, have a look at the :dox:`ArgumentParser` class.
+
+The Version Update Feature
+--------------------------
+
+With the seqan-2.3.0 release applications, using the :dox:`ArgumentParser`, check SeqAn servers for version updates. The functionality helps getting new versions out to users faster. It is also used to inform application developers of new versions of the SeqAn library which means that applications ship with less bugs.
+
+For example, you might get a message like this:
+
+.. code-block:: console
+
+  [APP INFO] :: There is a newer version of this application available.
+  [APP INFO] :: If this app is developed by SeqAn, visit www.seqan.de for updates.
+  [APP INFO] :: If you don't want to recieve this message again set --version_check OFF\n\n";
+
+The process of checking for a new version happens at most once a day. During this process the following information is transmitted to the servers:
```

Comment:
Also note that there might be a small delay for the server request, but that we use an internal timeout. Just that this is only relevant for performance sensible measurements.